### DISCUSSION TITLE
Contoh algoritma

### DISCUSSION THREAD

**Discussion by nlhmhs** [nlhmhs/Algoritma#3] [General]
Berikan saya code contoh sederhanya dalam kehidupan?

**Comment by dhfai** [comment]
Berikut adalah beberapa contoh algoritma yang diterapkan dalam kehidupan sehari-hari:

### 1. Algoritma Memasak Nasi

**Langkah-langkah:**
1. Ukur beras yang ingin dimasak.
2. Cuci beras hingga air cucian bening.
3. Masukkan beras ke dalam rice cooker.
4. Tambahkan air sesuai takaran (biasanya 1:1,5).
5. Tutup rice cooker dan nyalakan.
6. Tunggu hingga rice cooker selesai memasak (indikator berubah dari "cook" ke "warm").
7. Biarkan nasi selama 5-10 menit sebelum dibuka agar matang sempurna.
8. Nasi siap disajikan.

### 2. Algoritma Mencuci Pakaian Menggunakan Mesin Cuci

**Langkah-langkah:**
1. Pisahkan pakaian berdasarkan warna (putih, warna terang, warna gelap).
2. Masukkan pakaian ke dalam mesin cuci.
3. Tambahkan deterjen sesuai takaran.
4. Pilih mode pencucian yang sesuai (normal, heavy, delicate, dll.).
5. Nyalakan mesin cuci.
6. Tunggu hingga mesin cuci selesai (biasanya ada bunyi atau indikator yang menunjukkan selesai).
7. Keluarkan pakaian dan jemur atau masukkan ke dalam pengering.

### 3. Algoritma Menghitung Total Belanja di Supermarket

**Langkah-langkah:**
1. Ambil keranjang atau troli belanja.
2. Pilih barang-barang yang ingin dibeli.
3. Catat atau ingat harga setiap barang.
4. Saat selesai, bawa barang ke kasir.
5. Kasir akan memindai barcode setiap barang.
6. Total harga barang akan ditampilkan di mesin kasir.
7. Bayar total belanja sesuai dengan yang tertera.
8. Simpan struk sebagai bukti pembayaran.

### 4. Algoritma Menyetel Alarm untuk Bangun Pagi

**Langkah-langkah:**
1. Tentukan waktu yang diinginkan untuk bangun.
2. Ambil ponsel atau jam alarm.
3. Buka aplikasi alarm atau gunakan tombol pada jam alarm.
4. Setel waktu alarm sesuai dengan waktu yang diinginkan.
5. Aktifkan alarm.
6. Letakkan ponsel atau jam di tempat yang mudah dijangkau.
7. Tidur.
8. Ketika alarm berbunyi, bangun dari tempat tidur dan matikan alarm.

### 5. Algoritma Membuat Kopi

**Langkah-langkah:**
1. Siapkan alat dan bahan (kopi bubuk, air panas, cangkir, sendok, gula/susu jika diinginkan).
2. Panaskan air hingga mendidih.
3. Masukkan satu atau dua sendok kopi bubuk ke dalam cangkir.
4. Tuangkan air panas ke dalam cangkir.
5. Aduk hingga kopi tercampur rata.
6. Tambahkan gula atau susu jika diinginkan.
7. Aduk kembali hingga rata.
8. Kopi siap dinikmati.

Semua contoh di atas menunjukkan bagaimana algoritma sederhana dapat diterapkan dalam kegiatan sehari-hari. Setiap algoritma terdiri dari langkah-langkah yang harus diikuti untuk mencapai tujuan tertentu.
